great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -531,10 +538,20 @@ def _get_bootstrap_estimate(
         else:
             n_resamples = num_bootstrap_samples
 
+        # Obtain bootstrap_random_seed override from "rule state" (i.e., variables and parameters); from instance variable otherwise.
+        random_seed: Optional[int] = get_parameter_value_and_validate_return_type(
```

Comment:
@NathanFarmer Small request -- please make the variable name the same as that quoted in the comment -- just like in all other uses of `get_parameter_value_and_validate_return_type()` and the respective comments above that method call.  Thanks!